Smartling.API/Dashboard/DashboardApiClient.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Smartling.Api.Authentication;
+using Smartling.Api.Model;
+using System.Security.Authentication;
+
+namespace Smartling.Api.Dashboard
+{
+    public class DashboardApiClient : ApiClientBase
+    {
+        private readonly IAuthenticationStrategy _auth;
+        private const string DashboardUrl = "/translator-dashboard-api/v2/jobs";
+
+        public DashboardApiClient(IAuthenticationStrategy auth)
+        {
+            _auth = auth;
+        }
+
+        public virtual DashboardData GetDashboardData()
```

Comment:
Do we override it? mock in tests? Then let's don't mark it `override` if we don't have such plans